src/renderPreview/convertMd.ts

```diff
@@ -1,10 +1,22 @@
 import markdownIt, { Options } from 'markdown-it'
 import Renderer from 'markdown-it/lib/renderer'
 import markdownItPandoc from 'markdown-it-pandoc'
-import { Doc } from '../appState/AppState'
+import { Doc, Settings } from '../appState/AppState'
 
-const mdItPandoc = markdownItPandoc(markdownIt())
-const defaultImageRender = mdItPandoc.renderer.rules.image
+// Initialize markdown-it instance
+const md = markdownIt()
+let mdItPandoc: any
+let defaultImageRender: any
+// let mdItPandoc = markdownItPandoc(md)
+// let defaultImageRender = mdItPandoc.renderer.rules.image as Renderer.RenderRule
```

Comment:
I was figuring out how to load the settings but not initialize twice. This is working. Will remove the commented code.